overlord/fdestate/fdestate.go

```diff
@@ -367,6 +367,35 @@ func updateParameters(st *state.State, role string, containerRole string, bootMo
 	return nil
 }
 
+func getParameters(st *state.State, role string, containerRole string) (bootModes []string, models []secboot.ModelForSealing, tpmPCRProfile []byte, err error) {
```

Comment:
we have the manager now where get/updateParameters are called? can they become simply method on it?